### DISCUSSION TITLE
AxwayAspectJ-1.0.jar update on Axway Version :7.7.20230228

### DISCUSSION THREAD

**Discussion by priyanka-199100** [community/community#59662] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Issues

### Body

AxwayAspectJ-1.0.jar: We were using this code for Axway API Gateway to get features of One Agent SDK in particular, which connects Pure Paths.
Now, the issue is this code no longer supported after the upgrade of  Axway Version :7.7.20230228, How ever it supports below Jar file.
aspectjweaver-1.9.6.jar
oneagent-sdk-1.8.0.jar
Configured the JVM
But what we are missing : Copy dynatrace-aspectj-x.x.x.jar file to apigateway/ext/lib

Can someone suggest, if there is a new version that can be delivered as this ones seems to have some issue to connect.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by felipeosantos** [comment]
This package dynatrace-aspectj-x.x.x.jar is build result from projetc https://github.com/Axway-API-Management-Plus/dynatrace-apim-oneagent.git

**Comment by felipeosantos** [comment]
> ### Select Topic Area
> General
> 
> ### Feature Area
> Issues
> 
> ### Body
> AxwayAspectJ-1.0.jar: We were using this code for Axway API Gateway to get features of One Agent SDK in particular, which connects Pure Paths. Now, the issue is this code no longer supported after the upgrade of Axway Version :7.7.20230228, How ever it supports below Jar file. aspectjweaver-1.9.6.jar oneagent-sdk-1.8.0.jar Configured the JVM But what we are missing : Copy dynatrace-aspectj-x.x.x.jar file to apigateway/ext/lib
> 
> Can someone suggest, if there is a new version that can be delivered as this ones seems to have some issue to connect.
> 
> ### Guidelines
> * [x]  I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

This package dynatrace-aspectj-x.x.x.jar is build result from projetc https://github.com/Axway-API-Management-Plus/dynatrace-apim-oneagent.git
